pom.xml

```diff
@@ -620,6 +612,13 @@
         <version>1.53.0</version>
       </dependency>
 
+      <dependency>
+        <groupId>com.google.auto.value</groupId>
+        <artifactId>auto-value</artifactId>
+        <version>1.5</version>
+        <scope>provided</scope>
```

Comment:
auto-value is already defined higher up in the pom with version 1.6.3. Does it need to be 1.5?